pom.xml

```diff
@@ -264,6 +264,11 @@
       <artifactId>aws-java-sdk-sqs</artifactId>
       <version>1.11.68</version>
     </dependency>
+    <dependency>
+      <groupId>me.tongfei</groupId>
+      <artifactId>progressbar</artifactId>
+      <version>0.6.0</version>
+    </dependency>
```

Comment:
`test` scope?

Reply:
tried that, unfortunately that seemed to dump dependencies only into ~.m2 and it became super hard to run.